app/services/hmpps_api/prison_api/prison_api_base.rb

```diff
@@ -1,18 +1,18 @@
 # frozen_string_literal: true
 
-module Nomis
-  module Elite2
+module HmppsApi
+  module PrisonApi
     ApiPaginatedResponse = Struct.new(:total_pages, :data)
 
-    module Elite2Api
+    module PrisonApiBase
```

Comment:
personally I wouldn't call this PrisonApiBase because 
(a) it's a mixin 
(b) The term 'base' implies some magic base class

maybe PrisonApiClient?